requirements.txt

```diff
@@ -10,6 +10,7 @@ flup-py3==1.0.3
 Genshi==0.7.7
 gunicorn==20.1.0
 httpx==0.24.1
+ijson==3.2.3
```

Comment:
Not thrilled to see a new dependency, as we're not great at keeping everything up-to-date...

Reply:
Agreed; it was essential here, since otherwise the "dry-run" would take minutes to actually output the first line, defeating the purpose of a dry run. Also thought it would be handy in general; we do a lot of data things, so being able to stream JSON is super handy!